src/Orleans.SDK.targets

```diff
@@ -30,15 +30,14 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </When>
     <Otherwise>
       <PropertyGroup>
-        <CodeGenToolExe>$(CodeGenToolExeDir)\ClientGenerator.exe</CodeGenToolExe>
+        <CodeGenToolExe>$(CodeGenToolExeDir)/ClientGenerator.exe</CodeGenToolExe>
       </PropertyGroup>
     </Otherwise>
   </Choose>
 
   <!-- This target is run just before Compile for an Orleans Grain Interface Project -->
   <Target Name="OrleansCodeGeneration"
-          AfterTargets="BeforeCompile"
-          BeforeTargets="CoreCompile"
+          BeforeTargets="AssignTargetPaths"
```

Comment:
Is this a target that comes in all MSBuild projects or is it custom-defined? Because this has to run in a plain .NET project